src/avatars/avatars.module.ts

```diff
@@ -12,7 +12,12 @@ import { AvatarsService } from './avatars.service';
     TypeOrmModule.forFeature([Avatar]),
     MulterModule.register({
       storage: diskStorage({
-        destination: join(__dirname, '/images'),
+        destination: join(
+          __dirname,
+          process.env.NODE_ENV === 'test'
+            ? '../../test/uploads/avatars'
```

Comment:
This isn't quite right. Some avatars are used in tests, e.g. testing uploding images for avatar. And those images, which are exlusively used for testing, should be placed in a discrete directory.